tests/acceptance/features/serviceAvailability/serviceAvailabilityCheck.feature

```diff
@@ -0,0 +1,133 @@
+Feature: service health check
+
+
+  Scenario: health check defauts service
+    When a user requests these endpoints:
+      | endpoint                                        | service            | comment |
+      # | %base_url_without_scheme_and_port%:9197/healthz | activitylog        | #get 500 |
+      # | %base_url_without_scheme_and_port%:9297/healthz | antivirus          | checked in apiAntivirus suite |
```

Comment:
keep it as comment or remote it?
```suggestion
```

Reply:
I wanted to have list of all services here. and indicate that some are #extra (not running by default), some are checked in another suites